test/e2e/network/ingress.go

```diff
@@ -153,6 +154,29 @@ var _ = SIGDescribe("Loadbalancing: L7", func() {
 			// framework.ExpectNoError(jig.verifyURL(fmt.Sprintf("https://%v/", ip), "", 30, 1*time.Second, httpClient))
 		})
 
+		It("multicluster ingress should get instance group annotation", func() {
+			name := "echomap"
+			jig.CreateIngress(filepath.Join(framework.IngressManifestPath, "http"), ns, map[string]string{
+				"kubernetes.io/ingress.class": "gce-multi-cluster",
+			}, map[string]string{})
+
+			By(fmt.Sprintf("waiting for Ingress %s to come up", name))
+			pollErr := wait.Poll(2*time.Second, framework.IngressReqTimeout, func() (bool, error) {
```

Comment:
We should probably set another timeout instead of using `IngressReqTimeout` :)
```
	// IngressReqTimeout is the timeout on a single http request.
	IngressReqTimeout = 10 * time.Second
```

Reply:
Updated to 
```LoadBalancerPollTimeout  = 15 * time.Minute```